ceph.spec.in

```diff
@@ -652,6 +658,10 @@ fi
 %{_initrddir}/ceph
 %if 0%{?_with_systemd}
 %{_tmpfilesdir}/%{name}.conf
+%{_unitdir}/ceph-mds@.service
+%{_unitdir}/ceph-mon@.service
+%{_unitdir}/ceph-osd@.service
+%{_unitdir}/ceph.target
```

Comment:
We're adding unit files here, but not adding the `ceph-radosgw@.service` one later on